test cases/linuxlike/13 cmake dependency/meson.build

```diff
@@ -9,6 +9,12 @@ endif
 # Zlib is probably on all dev machines.
 
 dep = dependency('ZLIB', version : '>=1.2', method : 'cmake')
+
+# Temp fix for known failure
+if '#define' in dep.version()
```

Comment:
Would the better fix be something along the lines of:

```meson
if cmake.version().version_compare('<= 3.27.4) and def.version().version_compare('>= 1.3')
```

Reply:
How would `'1.3.#define ZLIB_VERSION "1.3"'.version_compare()` be able to work?

Reply:
Well, we really want to find the cmake version with meson, not cmake...

Reply:
Oh you're right. Remove the second portion of the `and`

Reply:
And maybe throw a comment referencing the CMake issue with the url

Reply:
Depends if they'll backport the fix instead of bumping to 3.27. Not sure you can do `version_compare('>= 1.3')` with a string that contains bogus `#define`. I think this check is fine, but let's see if it pass on CI.

Reply:
If the fix is backported then that won't cause the testsuite to fail, so I don't think it matters that we'd skip it.

Reply:
As mentioned in IRC I'd prefer a version check to avoid the situation where cmake breaks again and nobody notices because it is skipped.

Reply:
Updated. I think everybody will be happy now!